includes/Classifai/Services/LanguageProcessing.php

```diff
@@ -75,6 +77,24 @@ public function register_endpoints() {
 				'permission_callback' => [ $this, 'generate_post_excerpt_permissions_check' ],
 			]
 		);
+
+		register_rest_route(
+			'classifai/v1',
+			'synthesize-speech/(?P<id>\d+)',
+			[
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'synthesize_speech_from_text' ),
+				'args'                => array(
+					'id' => array(
+						'required'          => true,
+						'type'              => 'integer',
+						'sanitize_callback' => 'absint',
```

Comment:
@Sidsector9 WordPress handles sanitization of `type=number` request param ([Code](https://github.com/WordPress/WordPress/blob/996fe7077e2da39a1c7a795e414c2f8d411a7831/wp-includes/rest-api.php#L2791))

Do you think we can remove `sanitize_callback` param?

Reply:
Two things I'd mention:

1. This is my personal opinion so feel free to ignore but I like being verbose in these things rather than rely on the default (even if we run the same code as the default), just so it's clear at a glance that we are sanitizing things here
2. The default sanitization here is `return (int) $value` which ensures the `$value` is always an integer but will allow negative numbers. Using `absint` ensures we never have a negative number here

Reply:
@dkotter I agree with you. It is ok to use `absint`.

Reply:
@dkotter in this specific case, @ravinderk's suggestion will work fine because the regex `\d+` will not allow negative numbers and it will return a "No route was found" error.

Would you recommend removing the `absint` sanitisation checks?